datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_traps_db.py

```diff
@@ -128,7 +129,10 @@ def generate_traps_db(mib_sources, output_dir, output_file, output_format, no_de
             os.mkdir(mibs_sources_dir)
 
         mib_sources = (
-            sorted(set([os.path.abspath(os.path.dirname(x)) for x in mib_files if os.path.sep in x])) + mib_sources
+            sorted(
+                set([pathlib.Path(os.path.abspath(os.path.dirname(x))).as_uri() for x in mib_files if os.path.sep in x])
```

Comment:
Wondering if we can do something like this only using pathlib (not tested) `pathlib.Path(x).absolute().as_uri()`:

```suggestion
                set([pathlib.Path(x).absolute().as_uri() for x in mib_files if os.path.sep in x])
```

Reply:
Way better thanks!

Reply:
You also need to get the dir name of the path in this example

Reply:
Also looks like `.absolute` is undocumented and under-tested in python3.8. `.resolve` might be a better candidate: https://docs.python.org/3.8/library/pathlib.html#pathlib.Path.resolve

Reply:
Just found out that click is already resolving path [here](https://github.com/DataDog/integrations-core/blob/296a817476ba75e7cd4e69837838b129767ac466/datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_traps_db.py#L78) : `(resolve_path=True)`, `mib_files` is already an array of absolute paths. Do we still need to `.resolve` or can we directly do `pathlib.Path(x).as_uri()` ?

Reply:
Ah good point, probably not needed indeed. Maybe something like `pathlib.Path(x).parent.as_uri()` then?

Reply:
Yep thats perfect thanks!

Reply:
This might create more work than needed actually so I'll leave the decision up to you. I'm fine with the PR as is now 👍